### ISSUE TITLE
Loading Animation

### ISSUE THREAD

**Issue by RishabhJain0721** [RishabhJain0721/Brain-Box#8]
Add a loading animation (a simple loader) when a user logs in.
You can refer to [https://mhnpd.github.io/react-loader-spinner/](url)

**Comment by ayesha-2101** [comment]
I am willing to work on this under JWOC. Please assign this to me @RishabhJain0721

**Comment by RishabhJain0721** [comment]
@ayesha-2101 Go on.

**Comment by RishabhJain0721** [comment]
@ayesha-2101 Are you still on it?

**Comment by Jenis07** [comment]
Can you please assign me this issue under JWOC?

**Comment by RishabhJain0721** [comment]
@Jenis07 go on.

**Comment by Jenis07** [comment]
![image](https://github.com/RishabhJain0721/Brain-Box/assets/100338684/56cf2f6e-875b-4f1a-91e8-831871770d7f)
sir is this good or i have to added on to this?

**Comment by RishabhJain0721** [comment]
@Jenis07 You don't have to call me sir. The loader you have made is quite large.
Please make it smaller and such that it rotates inside the button where login is written and obviously, this should work on click.

**Comment by Jenis07** [comment]
Yes it will work on the only click of the button this is just for the demo and okay i will set it to the into the button..
